tests/objects/test_label_structure.py

```diff
@@ -1207,28 +1262,300 @@ def test_audio_classification_can_be_added_edited_and_removed(ontology, empty_au
 def test_audio_object_can_be_added_edited_and_removed(ontology, empty_audio_label_row: LabelRowV2):
     label_row = empty_audio_label_row
     obj_instance = ObjectInstance(audio_obj_ontology_item)
-    obj_instance.set_for_frames(AudioCoordinates(), Range(start=0, end=1500))
+    obj_instance.set_for_frames(AudioCoordinates(range=[Range(start=0, end=1500)]))
     range_list = obj_instance.range_list
+    assert range_list is not None
     assert len(range_list) == 1
     assert range_list[0].start == 0
     assert range_list[0].end == 1500
 
     label_row.add_object_instance(obj_instance)
     assert len(label_row.get_classification_instances()) == 0
     assert len(label_row.get_object_instances()) == 1
-    obj_instance.set_for_frames(AudioCoordinates(), Range(start=2000, end=2499))
+    obj_instance.set_for_frames(AudioCoordinates(range=[Range(start=2000, end=2499)]), overwrite=True)
     range_list = obj_instance.range_list
+    assert range_list is not None
+    assert len(range_list) == 1
+    assert range_list[0].start == 2000
+    assert range_list[0].end == 2499
+
+    obj_instance.remove_from_frames(frames=0)
+    range_list = obj_instance.range_list
+    assert range_list is None
+
+    label_row.remove_object(obj_instance)
+    assert len(label_row.get_object_instances()) == 0
+
+
+def test_get_annotations_from_html_text_object(ontology) -> None:
+    now = datetime.datetime.now()
+
+    range_html=HtmlRange(
+        start=HtmlNode(
+            node="/html[1]/body[1]/div[5]/div[1]/div[1]/div[1]/div[1]/div[1]/div[1]/div[2]/div[1]/div[1]/main[1]/div[2]/section[1]/div[1]/div[1]/div[2]/div[1]/p[1]/text()[1]",
```

Comment:
nitpick: since there is no validation here maybe we can make this path realistic but a bit shorter?